src/Decompressor/DecompressorGzip.php

```diff
@@ -16,10 +16,12 @@ public function decompress(SplFileInfo $sourceFile): void
         try {
             $baseName = $sourceFile->getBasename('.gz');
             $destinationPath = $this->getDestinationPath($sourceFile);
-            (new Process(
-                'gunzip -c ' . escapeshellarg($sourceFile->getPathname()) .' > ' . escapeshellarg($destinationPath . '/' . $baseName)
-            ))
-                ->setTimeout(null)
+            $process = Process::fromShellCommandline(
```

Comment:
proč tohle místo třeba 
```php
new Process([
  'gunzip',
  '-c',
  $sourceFile->getPathname(),
  ' > ',
  $destinationPath . '/' . $baseName
])
```

u snappy to tak máš

Reply:
protoze ti to udela na kazdym prvku pole escapeshellarg, takze nemuzes pouzit `>`, tj. v pripade, ze chces pouzit redirection, tak musis pouzit `fromShellCommandLine` https://symfony.com/doc/current/components/process.html#using-features-from-the-os-shell

Reply:
jsem si to myslel, dík!

Reply:
Škoda, že neudělali něco jako Zend_Db_Expr, ale pro bash. že bys standardně posílal ty argumenty a ten obalený by to ignorovalo.